abci/tutorials/abci-v2-forum-app/test/find_user_test.go

```diff
@@ -46,16 +46,10 @@ func TestFindUserByName(t *testing.T) {
 		t.Fatalf("Failed to find user by name: %v", err1)
 	}
 
-	if foundUser1 == nil {
-		t.Fatalf("User not found")
-	}
-
-	if foundUser1.Name != "user2" {
-		// t.Fatalf("Expected user2, but got %s", foundUser.Name)
-		println("Expected user2, but got %s", foundUser1.Name)
-	}
-
-	if foundUser1.Name == "user2" {
+	if foundUser1 != nil {
```

Comment:
I know we prefer early returns but for some reason if I place the check for nil first and call fatalF it does not recognize it as termination, but still reports the 'Possible nil pointer dereference' error